public/app/features/trails/TrailStore/utils.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import { locationService } from '@grafana/runtime';
+import { Button, Stack } from '@grafana/ui';
+
+import { createSuccessNotification } from '../../../core/copy/appNotification';
+import { HOME_ROUTE } from '../shared';
+
+export function createBookmarkSavedNotification() {
+  const notification = createSuccessNotification('Exploration successfully saved');
+
+  const onClick = () => {
+    locationService.push(HOME_ROUTE);
+  };
+
+  notification.component = (
+    <Stack gap={2} direction={'row'}>
+      <div>
+        You can view bookmarks under <i>Explore &gt; Metrics</i>
```

Comment:
I would turn Explore Metrics in to a regular blue inline TextLink instead of using the button, feels more natural. And feels less like a big break in how we use toasts. 

If UX feels the button is essential (even though it is a new way to use toasts), please change it into a normal LinkButton (no need for the onClick then) and gives users the choice to CMD+click to open in new tab

Reply:
Hm unfortunately the blue and green don't really meet accessibility standards on light mode
<img width="966" alt="Screenshot 2024-03-22 at 08 53 30" src="https://github.com/grafana/grafana/assets/5264102/3ec6939c-1631-4444-8151-432dcc2d9f4a">

Could we use the secondary button / solid styling instead? This is what is shown in the storybook component for the alerts, and the figma component for the toast.
<img width="405" alt="Screenshot 2024-03-22 at 08 56 37" src="https://github.com/grafana/grafana/assets/5264102/b7823ef0-2576-4370-9006-b2266729a783">

Reply:
or just "Bookmark saved"

Reply:
Yeah we can use "Bookmark created"

Reply:
lgtm!